packages/manager/modules/nasha/src/dashboard/partitions/partitions.routing.js

```diff
@@ -1,48 +1,64 @@
-import { STATE_NAME } from './partitions.constants';
+import { createTaskTrackerStateOptions } from '../../components/task-tracker';
+import {
+  INSTANCE_STATE_NAME,
+  STATE_NAME,
+  TASK_TRACKER_STATE_NAME,
+} from './partitions.constants';
 
 export default /* @ngInject */ ($stateProvider) => {
   const goToPagePartitionResolve = [
     { id: '', name: '' },
-    { id: 'access', name: 'Access' },
+    { id: 'accesses', name: 'Accesses' },
     { id: 'snapshots', name: 'Snapshots' },
-  ].reduce((resolves, { id, name }) => {
+  ].reduce((resolve, { id, name }) => {
     const resolveName = `goToPagePartition${name}`;
-    const stateId = id ? `.${id}` : '';
+    const stateName = `nasha.dashboard.partition${id ? `.${id}` : ''}`;
     return {
-      ...resolves,
+      ...resolve,
       [resolveName]: /* @ngInject */ ($state, serviceName) => (partitionName) =>
-        $state.go(`nasha.dashboard.partition${stateId}`, {
+        $state.go(stateName, {
           serviceName,
           partitionName,
         }),
     };
   }, {});
 
   const goToTabPartitionsResolve = [
-    { id: 'create', name: 'Create' },
-    { id: 'delete', name: 'Delete' },
-    { id: 'edit-size', name: 'EditSize' },
-    { id: 'zfs-options', name: 'ZfsOptions' },
-  ].reduce(
-    (resolves, { id, name }) => ({
-      ...resolves,
-      [`goToTabPartitions${name}`]: /* @ngInject */ ($state, serviceName) => (
-        partition,
-      ) =>
-        $state.go(`${STATE_NAME}.${id}`, {
+    { id: 'create', name: 'Create', instance: false },
+    { id: 'delete', name: 'Delete', instance: true },
+    { id: 'edit-size', name: 'EditSize', instance: true },
+    { id: 'zfs-options', name: 'ZfsOptions', instance: true },
+  ].reduce((resolve, { id, name, instance }) => {
+    const resolveName = `goToTabPartitions${name}`;
+    const stateName = `${instance ? INSTANCE_STATE_NAME : STATE_NAME}.${id}`;
+    return {
+      ...resolve,
+      [resolveName]: /* @ngInject */ ($state, serviceName) => (partition) =>
+        $state.go(stateName, {
           serviceName,
-          partition,
           partitionName: partition?.partitionName,
+          partition,
         }),
-    }),
-    {},
-  );
+    };
+  }, {});
 
   $stateProvider.state(STATE_NAME, {
     url: '/partitions',
     component: 'nashaDashboardPartitions',
     resolve: {
       breadcrumb: () => null,
+      close: /* @ngInject */ (goBack, trackTasks) => ({
+        tasks,
+        partitionName,
+        error,
+      } = {}) =>
+        tasks
+          ? trackTasks({ tasks, partitionName })
+          : goBack({ stateName: STATE_NAME, error }),
+      partitionsHref: /* @ngInject */ ($state, serviceName) => () =>
+        $state.href(STATE_NAME, { serviceName }),
+      trackTasks: /* @ngInject */ ($state) => (params) =>
+        $state.go(TASK_TRACKER_STATE_NAME, params),
```

Comment:
**nitpick**: function name suggests it does something related to tracking while it does a state navigation. Can you rename according to its responsibility?